firefox_cloud_services_PrivacyNotice/ta.md

```diff
@@ -7,7 +7,7 @@
 
 ## உங்களுக்கு தெரிய வேண்டியவைகள்:
 
-நீங்கள் பயன்படுத்தும் சேவைகளின் அம்சங்களைப் பொறுத்து எங்களுக்கு பல்வேறு வகையான தரவை அனுப்புகிறீர்கள். மேலும் அறிக.
+நீங்கள் பயன்படுத்தும் சேவைகளின் அம்சங்களைப் பொறுத்து எங்களுக்கு பல்வேறு வகையான தரவை அனுப்புகிறீர்கள்.
```

Comment:
I can vouch that this is perfect. How on earth do you know so many languages? :O @kyoshino